var/spack/repos/builtin/packages/py-gym/package.py

```diff
@@ -14,16 +14,17 @@ class PyGym(PythonPackage):
     environments."""
 
     homepage = "https://github.com/openai/gym"
-    pypi = "gym/0.18.0.tar.gz"
+    pypi = "gym/gym-0.18.0.tar.gz"
 
+    version('0.19.0', sha256="940069b983806e1ccc400fa6d47b4e34e462accf6a4fb0acb0a5e509ad0f502d")
     version("0.18.0", sha256="a0dcd25c1373f3938f4cb4565f74f434fba6faefb73a42d09c9dddd0c08af53e")
 
     depends_on("python@3.6:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
-    depends_on("py-scipy", type=("build", "run"))
-    depends_on("py-numpy@1.10.4:", type=("build", "run"))
+    depends_on("py-scipy", type=("build", "run"), when="@0.18.0")
+    depends_on("py-numpy@1.10.4:", type=("build", "run"), when="@0.18.0")
+    depends_on("py-numpy@1.18.0:", type=("build", "run"), when="@0.19.0")
     depends_on("py-pyglet@1.4.0:1.5.0", type=("build", "run"), when="@0.18.0")
-    depends_on("py-pyglet@1.4.0:1.5.15", type=("build", "run"), when="@0.18.1")
-    depends_on("pil@:8.2.0", type=("build", "run"), when="@0.18.1")
     depends_on("pil@:7.2.0", type=("build", "run"), when="@0.18.0")
-    depends_on("py-cloudpickle@1.2.0:1.6", type=("build", "run"))
+    depends_on("py-cloudpickle@1.2.0:1.6", type=("build", "run"), when="@0.18.0")
+    depends_on("py-cloudpickle@1.2.0:1.7.0", type=("build", "run"), when="@0.19.0")
```

Comment:
```suggestion
    depends_on("py-cloudpickle@1.2.0:1.6", type=("build", "run"))
```
This dep hasn't changed. It's `<1.7`, which is `:1.6` in Spack (version ranges are inclusive in Spack).

Reply:
O:-)

I know how version ranges work... I do not know why I saw a difference in the dep version for 0.19...

Sorry!